bootstrap/httpserver/server_handlers_test.go

```diff
@@ -27,244 +27,244 @@ import (
 
 func TestHttpServer_SendTransaction_Basic(t *testing.T) {
 	with.Logging(t, func(parent *with.LoggingHarness) {
-		h := newHarness(parent)
+		withServerHarness(parent, func(h *harness) {
```

Comment:
out of curiosity, why did you change the pattern to accept a func taking the harness?

Reply:
because the harness now needs to be shut down, and that way the wrapper handles it instead of the tests